src/makeCalculator.js

```diff
@@ -37,7 +37,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(command, number) {
+      this.result = command(this.result, number);
```

Comment:
* Consider binding context here
* Consider implementing a check if the command is a function and a number number type before calling.